spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -76,8 +76,8 @@ public TaskDefinitionRepository taskDefinitionRepository() {
 	}
 
 	@Bean
-	public ModuleRegistry moduleRegistry() {
-		return new StubModuleRegistry();
+	public ModuleRegistry moduleRegistry(RedisConnectionFactory redisConnectionFactory) {
+		return new RedisModuleRegistry(redisConnectionFactory);
```

Comment:
At some point, we will probably add a "store" property that could apply to the `ModuleRegistry` and the `StreamDefinitionRepository` (or separate properties for each).